src/app/components/Curation/index.tsx

```diff
@@ -63,14 +67,56 @@ export default ({
   });
 
   const GridComponent = getGridComponent(componentName);
+  const environmentIsLive = isLive();
 
   const isFirstCuration = position === 0;
   const curationSubheading = title || topStoriesTitle;
   const id = idSanitiser(curationSubheading);
 
+  const summaryHasRequiredValues = (summary: BillboardSummary) =>
+    summary.description && summary.link && summary.imageUrl && summary.imageAlt;
+
   switch (componentName) {
     case NOT_SUPPORTED:
       return null;
+    case BILLBOARD: {
+      if (summaries.length > 0) {
+        // block scope for new billboard summaries type
+        const billboardSummaries = summaries as BillboardSummary[];
+
+        if (environmentIsLive) {
+          return (
+            <MessageBanner
+              heading={title}
+              description={billboardSummaries[0].description}
+              link={billboardSummaries[0].link}
+              linkText={billboardSummaries[0].title}
+              image={billboardSummaries[0].imageUrl}
+              eventTrackingData={{
+                componentName: `message-banner-${nthCurationByStyleAndProminence}`,
+                detailedPlacement: `${position + 1}`,
+              }}
+            />
+          );
+        }
+
+        return summaryHasRequiredValues(billboardSummaries[0]) ? (
+          <Billboard
+            heading={title}
+            description={billboardSummaries[0].description}
+            link={billboardSummaries[0].link}
+            image={billboardSummaries[0].imageUrl}
+            eventTrackingData={{
+              componentName: `billboard-${nthCurationByStyleAndProminence}`,
+              detailedPlacement: `${position + 1}`,
+            }}
+            showLiveLabel={billboardSummaries[0].isLive}
+            altText={billboardSummaries[0].imageAlt}
+          />
+        ) : null;
+      }
+      return null;
+    }
```

Comment:
I wonder if we can simplify this, and make it reusable for the message banner code below. 

I also don't think it warrants having a new type for the `BillboardSummary`

I am thinking we should get the first summary outside of the switch / case statement:

`const [firstSummary] = summaries || [];` (Or whatever we need to do to stop TS errors, without ignoring / suppressing them)

Then destructure the values we need for the message banner / billboard
`const { description, link, imageAlt, imageUrl, isLive, title: linkText } = firstSummary;` (we could also consider renaming them at this point, if we think they are not descriptive enough

We could then also use these same values in the `MESSAGE_BANNER` case statement.

```suggestion
      if (summaries.length > 0) {
        if (environmentIsLive) {
          return (
            <MessageBanner
              heading={title}
              description={description}
              link={link}
              linkText={linkText}
              image={imageUrl}
              eventTrackingData={{
                componentName: `message-banner-${nthCurationByStyleAndProminence}`,
                detailedPlacement: `${position + 1}`,
              }}
            />
          );
        }

        return summaryHasRequiredValues(billboardSummaries[0]) ? (
          <Billboard
            heading={title}
            description={description}
            link={link}
            image={imageUrl}
            eventTrackingData={{
              componentName: `billboard-${nthCurationByStyleAndProminence}`,
              detailedPlacement: `${position + 1}`,
            }}
            showLiveLabel={isLive}
            altText={imageAlt}
          />
        ) : null;
      }
      return null;
    }
```

Reply:
Thanks,  I'm confused by the commit suggestion because it still has `summaryHasRequiredValues(billboardSummaries[0])` within the return, was that a typo?

Reply:
I think we might need to rejig what's there - don't take my commit suggestion as gospel! If you're around on Monday we could try and pair on it / I can explain what I mean?

Reply:
I'll be in on Mon so can chat through it but working on it atm if you're free for a chat. No worries if you're busy though!

I've half implemented what I think is the suggestion but it still presents the problem of conflicting needs based on the types (with `description, link, imageUrl` etc. In a message banner, the link/image etc are not mandatory and so it's not immediately clear how to enforce that on a billboard using the same `summary` `type` structure